### PR TITLE
Add a test that the beam can be steered fast enough

### PR DESCRIPTION
**Pull request by bmerry**
Closes NGC-1259.

Checklist (if not applicable, edit to add `(N/A)` and mark as done):

- [x] (N/A) If dependencies are added/removed: update `setup.cfg` and `.pre-commit-config.yaml`
- [x] (N/A) If modules are added/removed: use `sphinx-apidoc -efo doc/ src/` to update files in `doc/`
- [x] Ensure copyright notices are present and up-to-date
- [x] If qualification tests are changed: attach a sample qualification report: [report.pdf](https://github.com/ska-sa/katgpucbf/files/14521338/report.pdf)
- [x] (N/A) If design has changed: ensure documentation is up to date
- [x] (N/A) If ICD-defined sensors have been added: update `fake_servers.py` in katsdpcontroller to match

### CODE REVIEW

File: qualification/tied_array_channelised_voltage/test_delay.py [hunk 1]
```diff
@@ -207,3 +209,40 @@ async def test_delay(
...
+    Verification method
+    -------------------
+    Verified by means of test. Measure the time taken to issue
+    ``?beam-delays`` requests for all beams.
+    """
+    receiver = receive_tied_array_channelised_voltage
+    client = cbf.product_controller_client
+
+    # Use random values just because they'll have lots of significant digits
+    # and hence take longer to format/parse.
```
Comment (amishatishpatel) [inline qualification/tied_array_channelised_voltage/test_delay.py:231]: Noted.

File: qualification/tied_array_channelised_voltage/test_delay.py [hunk 2]
```diff
@@ -207,3 +209,40 @@ async def test_delay(
...
+    start = time.monotonic()
+    pdf_report.detail(f"Start time is {start:.6f}.")
+    for stream_name, stream_params in zip(receiver.stream_names, params):
+        await client.request("beam-delays", stream_name, *stream_params)
+        pdf_report.detail(f"Set delays on {stream_name} to {stream_params}")
+    stop = time.monotonic()
+    pdf_report.detail(f"Stop time is {stop:.6f}.")
+    elapsed = stop - start
+    pdf_report.detail(f"Elapsed time is {elapsed:.6f}.")
+    assert elapsed < 5.0
```
Comment (amishatishpatel) [inline qualification/tied_array_channelised_voltage/test_delay.py:248]: From the requirement text
> [...] and the execution time delay is no more than 2 seconds.

Not to be so blunt, but you've got 5.0 here? Let me know if there's a separate doc/ECP/something I should look at that overrides the 2s mentioned in the REQ.

**Review by amishatishpatel — COMMENTED** [review]
As neat as this test is, just the one (slightly major) comment for clarification before I can explicitly approve.

File: qualification/tied_array_channelised_voltage/test_delay.py (hunk 2 above)
Comment (bmerry) [reply qualification/tied_array_channelised_voltage/test_delay.py:248]: I see I linked to the wrong requirement. It should have been CBF-REQ-0076:

> The CBF shall execute all of the commands required to re-point a TA beam
utilising H and V polarisation of all available antennae in less than 5
seconds.

Comment (amishatishpatel) [reply qualification/tied_array_channelised_voltage/test_delay.py:248]: Aha, yeah the whole context behind REQ-0077 seemed a bit odd here (with the tracking of the request 200ms _before_ execution time).

Either way, please and thank you.

**Review by amishatishpatel — APPROVED** [review]

**Pull request merged**